src/simulation/routing/router.rs

```diff
@@ -1,459 +1,12 @@
-use crate::simulation::network::routing_kit_network::RoutingKitNetwork;
```

Comment:
I decided to put this former code into rust_road_router.rs. This file does only contain the trait.